src/wallet.cpp

```diff
@@ -2713,11 +2713,25 @@ bool CWallet::CreateTransaction(const vector<pair<CScript, CAmount> >& vecSend,
                     // TODO: pass in scriptChange instead of reservekey so
                     // change transaction isn't always pay-to-pivx-address
                     CScript scriptChange;
+                    bool combineChange = false;
 
                     // coin control: send change to custom address
-                    if (coinControl && !boost::get<CNoDestination>(&coinControl->destChange))
+                    if (coinControl && !boost::get<CNoDestination>(&coinControl->destChange)){
                         scriptChange = GetScriptForDestination(coinControl->destChange);
 
+                        vector<CTxOut>::iterator it = txNew.vout.begin();
+                        for(; it != txNew.vout.end();){
```

Comment:
Nit: need a space between the closing `)` of the conditional and the opening bracket `{` 
Also a space between `for` and the opening `(` of the conditional.